CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to Distil will be documented in this file. This file follows the *[Keep a CHANGELOG](http://keepachangelog.com/)* standards.
+
+## 0.1.0-alpha.1 - 2017-09-06
```

Comment:
Let's just release 0.1.0 already. No need for an alpha release if we're actually going to use this.